README.md

```diff
@@ -56,12 +56,13 @@ Follow the three simple steps below to get Asterinas up and running.
 
 ```bash
 git clone https://github.com/ldos-project/asterinas
+cd asterinas
 ```
 
 2. Run a Docker container as the development environment.
 
 ```bash
-docker run -it --privileged --network=host --device=/dev/kvm -v $(pwd)/asterinas:/root/asterinas ldosproject/asterinas:0.15.2-20250613
+make docker
```

Comment:
I think people will need to customize this pretty often, so including the command here as an "advanced version" is probably a good idea.